src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
-
 spring.datasource.url=jdbc:postgresql://localhost:5432/mytest
-spring.datasource.username= postgres
-spring.datasource.password= user
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto = update
-server.port = 8081
+spring.datasource.username=postgres
+spring.datasource.password=user
```

Comment:
Just as a comment (no need to change anything 😅) remember that in general the credentials and sensitive info are set as environment variables/secrets so as to not push them to the repo :) for the training purposes this is fine, so don't worry 😄